src/chainparams.cpp

```diff
@@ -563,7 +563,7 @@ class CDevNetParams : public CChainParams {
         consensus.BIP65Height = 581885; // 00000000007f6655f22f98e72ed80d8b06dc761d5da09df0fa1dc4be4f861eb6
         consensus.BIP66Height = 330776; // 000000002104c8c45e99a8853285a3b592602a3ccde2b832481da85e9e4ba182
         consensus.powLimit = uint256S("0000ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
-        consensus.powLimitRandomX = uint256S("000fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
+        consensus.powLimitRandomX = uint256S("0000ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
         consensus.powLimitProgPow = uint256S("0000000fffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
         consensus.powLimitSha256 = uint256S("0000000fffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
```

Comment:
Given the current algorithm for adjusting the target:
```
    bnNew *= nActualTimespan;
    bnNew /= nTargetTimespan;
```
These, in combination with TargetSpacing define maximum `nDgwPastBlocks`.

I suggest one of 4 approaches (open to other ideas as well)
1) Note in CMainParams() that nDgwPastBlocks can't go over 127 (determined by 0xFFFF / 3 [the railed max] / 172 [the target spacing) = 127)
2) add an assert to check (max256 - powLimit) / 3 / targetspacing > nDgwPastBlocks
3) remove the overflow potential with bnNew *= (float)(actual/target)
4) remove the overflow potential with averaging the timespans (timespan/blocks)

Since it will work with the current settings; this is more of a nit